src/components/ReportsFilterPanel.vue

```diff
@@ -0,0 +1,188 @@
+      <div class="joined-container">
+        <label for="joined-after" class="col"
+          >Joined after
+          <input
+            id="joined-after"
+            type="date"
+            :value="joinedAfter"
+            @input="setJoinedAfter"
+          />
+        </label>
+
+        <label for="joined-before" class="col"
+          >Joined before
+          <input
+            id="joined-before"
+            type="date"
+            :value="joinedBefore"
+            @input="setJoinedBefore"
+          />
+        </label>
+      </div>
+    </div>
+
+    <div class="col">
+      <div class="session-range-container">
+        <label for="session-range-from" class="col"
+          >Session from
+          <input
+            id="session-range-from"
+            type="date"
+            :value="sessionRangeFrom"
+            @input="setSessionRangeFrom"
+          />
+        </label>
+
+        <label for="session-range-to" class="col"
+          >Session to
+          <input
+            id="session-range-to"
+            type="date"
+            :value="sessionRangeTo"
+            @input="setSessionRangeTo"
+          />
+        </label>
+      </div>
+    </div>
+
+    <div class="col">
+      <div>
+        <label for="student-partner-org" class="col"
+          >Student partner org
+          <v-select
+            id="student-partner-org"
+            class="student-partner-org"
+            :options="studentPartnerOrgs"
+            :searchable="true"
+            @input="setStudentPartnerOrg"
+          />
+        </label>
+
+        <div class="high-school">
+          <label for="high-school" class="col"
+            >High school
+            <school-list
+              :setHighSchool="setHighSchool"
+              placeholder="Search for a school"
+            />
+          </label>
+        </div>
+      </div>
+    </div>
+    <div class="col"></div>
+  </div>
+</template>
+
+<script>
+import SchoolList from "@/components/SchoolList";
+
+export default {
+  name: "FilterPanel",
+  components: { SchoolList },
+  props: {
+    joinedAfter: {
+      type: String,
+      required: true
+    },
+    joinedBefore: {
+      type: String,
+      required: true
+    },
+
+    setJoinedAfter: {
+      type: Function,
+      required: true
+    },
+    setJoinedBefore: {
+      type: Function,
+      required: true
+    },
+    sessionRangeFrom: {
+      type: String,
+      required: true
+    },
+    sessionRangeTo: {
+      type: String,
+      required: true
+    },
+    setSessionRangeFrom: {
+      type: Function,
+      required: true
+    },
+    setSessionRangeTo: {
+      type: Function,
+      required: true
+    },
+    studentPartnerOrg: {
+      type: String,
+      required: true
+    },
+    setStudentPartnerOrg: {
+      type: Function,
+      required: true
+    },
+    setHighSchool: {
+      type: Function,
+      required: true
+    }
+  },
+  computed: {
+    studentPartnerOrgs() {
+      return [];
```

Comment:
Is it okay to expose our partner orgs on GitHub? If so I'll populate them here

Reply:
I think it'd be better not to 1. expose them, and 2. have to maintain the list in multiple places

what about a server endpoint that returns the entire org manifest array? Something like that would probably be helpful for future admin tools (e.g. `admin/partners/student/list`)